lib/symbols.js

```diff
@@ -33,6 +33,8 @@ const wildcardFirstSym = Symbol('pino.wildcardFirst')
 const serializersSym = Symbol.for('pino.serializers')
 const wildcardGsym = Symbol.for('pino.*')
 const needsMetadataGsym = Symbol.for('pino.metadata')
+const bindingsSerializerSym = Symbol.for('pino.serializers.bindings')
+const levelSerializerSym = Symbol.for('pino.serializers.level')
```

Comment:
So the problem with this naming scheme is it's incongruous. 

`Symbol.for('pino.serializers')` is used to get the serializers for a given logger instance (http://getpino.io/#/docs/api?id=loggersymbolfor39pinoserializers39), when we configure serializer via opts we don't use a symbol.

`Symbol.for('pino.*')` is a global serializer, but it it's NOT pino.serializers.* 

So either these should be called 

* `Symbol.for('pino.bindings')` 
* `Symbol.for('pino.level')` 

OR we need to deprecate and break: 

* `Symbol.for('pino.*')` -> `Symbol.for('pino.serializers.*')`
* `Symbol.for('pino.serializers')` -> `Symbol.for('pino.serializers-acesssor')`  (?)

One other alternative is to use do associate the dollar sign `$` with serializer setters/config symbols:

* `Symbol.for('pino.*')` -> `Symbol.for('pino.$.*')`
* `Symbol.for('pino.serializers')` stays the same 
* `Symbol.for('pino.$.bindings')`
* `Symbol.for('pino.$.level')`

For now I'd go with the easy option:

* `Symbol.for('pino.bindings')` 
* `Symbol.for('pino.level')` 

And then we'll batch assess Symbol namespaces in pino next

Reply:
Another fundamental question is, should these be symbols. `pino.*` was deliberately provided as a symbol because we felt it could be a footgun and to show that it's distinct from normal serializers, in that it's one level above normal serializers.  These new serializers are even a level above that again. 

So we have three levels

* log base serializers (this PR)
* entire log object serializer  (`Symbol.for('pino.*'`)
* msg object key serializers (the original) 

I think we should have this API instead /cc @mcollina @jsumners 

```js
const pino = require('./')({
  superSerializers: {
    bindings (bindings) {
      return {
        process: { pid: bindings.pid },
        hostname: bindings.hostname
      }
    },
    level (label, number) {
      return {
        log: { level: label }
      }
    },
   '*' (obj) { // replaces Symbol.for('pino.*')
     obj.foo = 'whatever' 
     return obj
   }
  }
})
```

Reply:
If the benchmarks for these super serializers are slow then it should instead be:

```js
const pino = require('./')({
  [Symbol.for('pino.superSerializers')]: {
    bindings (bindings) {
      return {
        process: { pid: bindings.pid },
        hostname: bindings.hostname
      }
    },
    level (label, number) {
      return {
        log: { level: label }
      }
    },
   '*' (obj) { // replaces Symbol.for('pino.*')
     obj.foo = 'whatever' 
     return obj
   }
  }
})
```

Reply:
I'm not really a fan of exposing a client API through usage of symbols unless it is necessary to keep a serialized object clean. I think they should be used internally to give strong indication that anyone using them _must_ expect breakage at some point. Thus, we wouldn't have to issue a new major if we decide to change symbol descriptors.

Which is to say, I prefer https://github.com/pinojs/pino/pull/775#discussion_r381267993

Reply:
For me either is fine, probably I would choose a different name, such as `globalSerializers`.
Still, there is the "issue" when creating a child, where I need to override the parent's bindings serializer.

Reply:
We could even call it `fomatters`.
```js
const pino = require('./')({
  formatters: {
    bindings (bindings) {
      return {
        process: { pid: bindings.pid },
        hostname: bindings.hostname
      }
    },
    level (label, number) {
      return {
        log: { level: label }
      }
    },
   log (obj) {
     obj.foo = 'whatever' 
     return obj
   }
  }
})
```

Reply:
I'm happy with `formatters` :+1:

Reply:
I'm not sure that `log` is the right name or not

Reply:
https://github.com/pinojs/pino/blob/d237668d71e4e62224269eeef0b1b6dc4d668dbd/lib/levels.js#L119-L136